src/transformState.js

```diff
@@ -5,7 +5,46 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'clear': {
+        toClear();
```

Comment:
I don't think you really need to create separate functions here. It looks to me that it makes your code more difficult to read. Try it without additional functions

Reply:
maybe I will use them in the future - this functions, so I create them. for me when I write this was more understandable to work with separate function